civihr_employee_portal/civihr_employee_portal.module

```diff
@@ -135,8 +135,10 @@ function _setup_modals() {
  * @param string $resources the name of the script file (extension included)
  * @param string $extension
  * @param string $page
+ * @param array $jsOptions
```

Comment:
Since you've changed the params, you need to update the docblock as well

Reply:
fixed!